client/state/rewind/index.js

```diff
@@ -2,9 +2,14 @@
 /**
  * Internal dependencies
  */
-import { REWIND_STATE_UPDATE } from 'state/action-types';
+import { REWIND_ALERTS_UPDATE, REWIND_STATE_UPDATE } from 'state/action-types';
 import { keyedReducer } from 'state/utils';
 
+export const rewindAlertsItem = ( state = null, { type, alerts } ) =>
```

Comment:
Should we add tests for this?

Reply:
I recommend against it. it's insanely simple right now and not meant to be a significant part of this PR - just bare-minimum and anyway it should probably be replaced (with 100 other reducers) by some simpler abstraction that's well tested/testable

```js
rewindAlerts: keyedReducer( 'siteId', storeOn( REWIND_ALERTS_UPDATE, 'alerts' ) );
```

what tests would you write for this function?

Reply:
Makes sense. I know people like to add tests for reducers, but in this case it doesn't seem worth it yet.